### DISCUSSION TITLE
How do I check version of an unknown binary installation?

### DISCUSSION THREAD

**Discussion by ccaudle** [michaelwillis/dragonfly-reverb#163] [General]
I have a couple of machines with Dragonfly LV2 installed, and I do not remember for sure which version is installed.  
I checked the ttl files, and also ran strings on the so file, but did not see what look like a version string.  I think I should have 3.2.5 installed currently, and was trying to make sure before I replaced the files with 3.2.10.  I can just blindly replace but the attempt made me curious if there is a location with the version string stored so I can actually check in the future and not just blindly copy over a new version when I can't remember what I last installed.

**Comment by michaelwillis** [comment]
Click the question mark icon, it will show you the version and credits.
